src/Apps/Artist/Components/ArtistHeader/ArtistHeader2.tsx

```diff
@@ -303,4 +317,19 @@ const formatFollowerCount = (n: number) => {
   }
 }
 
+const parseVerifiedRepresentatives = (
+  representatives: ArtistHeader2_artist$data["verifiedRepresentatives"]
+) => {
+  return representatives.map((rep, index) => {
+    return (
+      <Fragment key={rep.slug}>
+        {index > 0 && ", "}
+        <RouterLink to={`/partner/${rep.slug}`} color="black100">
+          {rep.name}
+        </RouterLink>
+      </Fragment>
+    )
+  })
+}
```

Comment:
This function is unnecessary, just inline this. If you wanted to extract it would be a component and fragment container. But not needed in this case.